### PR TITLE
[MOSIP-33380] added helm for id-repository

### PR DESCRIPTION
**Pull request by Prafulrakhade**

### CODE REVIEW

File: .github/workflows/chart-lint-publish.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+jobs:
+  chart-lint-publish:
+    uses: mosip/kattu/.github/workflows/chart-lint-publish.yml@master
+    with:
+      CHARTS_DIR: ./helm
+      CHARTS_URL: https://mosip.github.io/mosip-helm
+      REPOSITORY: mosip-helm
+      BRANCH: gh-pages
+      INCLUDE_ALL_CHARTS: "${{ inputs.INCLUDE_ALL_CHARTS || 'NO' }}"
+      IGNORE_CHARTS: "${{ inputs.IGNORE_CHARTS ||'redis' }}"
```
Comment (ckm007) [inline .github/workflows/chart-lint-publish.yml:54]: is this needed here?

File: helm/credential/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## TL;DR
+
+```console
+$ helm repo add mosip https://mosip.github.io
+$ helm -n idrepo install my-release mosip/credential
+```
+## Prerequisites
+
+- Kubernetes 1.12+
```
Comment (ckm007) [inline helm/credential/README.md:13]: update the pre-requisites version

File: .github/workflows/chart-lint-publish.yml (hunk 1 above)
Comment (Prafulrakhade) [reply .github/workflows/chart-lint-publish.yml:54]: yes sir it needed for publishing the chart

File: helm/credential/README.md (hunk 2 above)
Comment (Prafulrakhade) [reply helm/credential/README.md]: updated

**Review by ckm007 — APPROVED** [review]

**Pull request merged**